modules/rest/openapi/openapi.yaml

```diff
@@ -61,6 +61,48 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /management/v1/cluster/topology/logical:
```

Comment:
just out of curiosity: why do we need this file if we use Swagger that generates schema?

Reply:
Got your point. This specification is autogenerated by micronaut-openapi generator. It is placed here to make access easier for third-party clients.

Reply:
So in theory if someone changed java code but forgot to run generator we can end up with two different schemas, right?